imagersite/templates/registration/registration_form.html

```diff
@@ -0,0 +1,69 @@
+{% extends 'imagersite/base.html' %}
+
+{% load static %}
+
+{% block component_css %}
+<link rel="stylesheet" href="{% static 'css/splash_page.css' %}">
+{% endblock %}
+
+{% block nav %}
+  <nav class="navbar navbar-dark navbar-fixed-top scrolling-navbar">
+    <button class="navbar-toggler hidden-sm-up" type="button" data-toggle="collapse" data-target="#collapseEx"><i class="fa fa-bars"></i></button>
+      <div class="container">
+          <div class="collapse navbar-toggleable-xs" id="collapseEx">
+              <a class="navbar-brand" href="/" target="_blank">PictureCrap</a>
+              <ul class="nav navbar-nav">
+                  <li class="nav-item">
+                      <a class="nav-link" href="#home">Home <span class="sr-only">(current)</span></a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="#best-features">Features</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="#examples-of-use">Examples</a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link" href="#testimonials">Testimonials</a>
+                  </li>
+              </ul>
+              <ul class="nav navbar-nav nav-flex-icons">
+                  <li class="nav-item">
+                      <a class="nav-link"><i class="fa fa-facebook"></i></a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link"><i class="fa fa-twitter"></i></a>
+                  </li>
+                  <li class="nav-item">
+                      <a class="nav-link"><i class="fa fa-instagram"></i></a>
+                  </li>
+              </ul>
+          </div>
+      </div>
+  </nav>
+{% endblock %}
+
+{% block body %}
+<div class="card">
+    <div class="card-block">
+        <div class="form-header blue-gradient">
+            <h3><i class="fa fa-user"></i> Register:</h3>
```

Comment:
There's already a registration form you can use that's provided by the django auth. You could pull it into this template with `{{ form }}`